test/gtest/type-builder.cpp

```diff
@@ -294,8 +294,8 @@ TEST_F(TypeTest, InvalidUnsharedSupertype) {
   TypeBuilder builder(2);
   builder[0] = Struct{};
   builder[1] = Struct{};
-  builder[0].setShared(false);
-  builder[1].setShared(true);
+  builder[0].setShared(Unshared);
+  builder[1].setShared(Shared);
```

Comment:
If we add overloads `setShared(bool)` then this could be a little shorter and I think just as unambiguous and clear, but I don't feel strongly.

Reply:
Same reason to avoid mixing bool and `Shareability` applies here.